### PR TITLE
backfill air table data

### PR DESCRIPTION
**Pull request by johndeyrup**
Job to backfill all data from the air table into postgreSQL. There are two extra fields we need to add from the air table. I didn't mess with the data in the availability and days open columns because the data was so unstructured it seemed better to just wait until the scrapers/volunteers could enter structured data. We could try to get some of the availability data with regexs if we want.

### CODE REVIEW

File: jobs/migration.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        county: airTableObject['County'],
+        latitude: airTableObject['Latitude'],
+        longitude: airTableObject['Longitude'],
+        lastupdated: airTableObject['Last Updated'],
+        website: airTableObject['Website'],
+        accessibility: airTableObject['Accessibility'] ? airTableObject['Accessibility'].replace('\n', '') : '', // to add
+        phone: airTableObject['Phone'], // to add
+        email: airTableObject['E-mail'] // to add
+    };
+}
```
Comment (jonhines) [inline jobs/migration.js:37]: this is very encouraging that this was so easy!

**Comment by jonhines** [comment]
👍

**Review by jonhines — APPROVED** [review]

**Comment by mzagaja** [comment]
It looks like this one was run on the vaccinatema database already. I think that's fine, and appreciate the documentation of the process here. I just want to confirm whether we want to designate/keep `vaccinatema` as our production DB going forward, and then maybe setup vaccinatema_staging or vaccinatema_development for testing against? Once we start using it live we're going to want to lock down the production DB from accidents.

**Comment by johndeyrup** [comment]
I would love a qa db for testing.

**Pull request merged**